server/src/main/java/org/jboss/as/server/ServerEnvironment.java

```diff
@@ -385,13 +395,13 @@ public ServerEnvironment(final String hostControllerName, final Properties props
             final ConfigurationFile.InteractionPolicy configInteractionPolicy, final LaunchType launchType,
             final RunningMode initialRunningMode, ProductConfig productConfig, long startTime, boolean startSuspended,
             String gitRepository, String gitBranch, String gitAuthConfiguration, String supplementalConfiguration) {
-        this(hostControllerName, props, env, serverConfig, configInteractionPolicy, launchType, initialRunningMode,
+        this(hostControllerName, props, env, serverConfig, configInteractionPolicy, launchType, initialRunningMode, Stream.STABLE,
```

Comment:
Is there a reason to define this as STABLE instead of just DEFAULT?

Reply:
Yeah - this should use DEFAULT.

Reply:
Nice catch.  Fixed.